src/main/scala/net.manub.embeddedkafka/schemaregistry/ops/SchemaRegistryOps.scala

```diff
@@ -50,13 +51,25 @@ trait SchemaRegistryOps {
                           avroCompatibilityLevel: AvroCompatibilityLevel =
                             AvroCompatibilityLevel.NONE,
                           properties: Properties = new Properties): RestApp = {
-    val server = new RestApp(schemaRegistryPort,
+
+    def findAvailablePort: Int = {
+      val server = new ServerSocket(0)
+      val port = server.getLocalPort
+      server.close()
+      port
+    }
+
+    val schemaRegistryRestAppPort =
+      if (schemaRegistryPort == 0) findAvailablePort else schemaRegistryPort
+
+    val server = new RestApp(schemaRegistryRestAppPort,
```

Comment:
`RunningSchemaRegistryOps.startSchemaRegistry` must now return an instance of `EmbeddedSR` with the actual port in use (as done [here](https://github.com/embeddedkafka/embedded-kafka/blob/master/embedded-kafka/src/main/scala/net/manub/embeddedkafka/ops/kafkaOps.scala#L86)).
Get the value from `restApp.app.restServer.getURI.getPort` (as done [here](https://github.com/embeddedkafka/embedded-kafka-schema-registry/blob/master/src/main/scala/net.manub.embeddedkafka/schemaregistry/EmbeddedKafka.scala#L38)).